README.md

```diff
@@ -0,0 +1,23 @@
+# Fairphone Clock Widget
+
+Clock widget originally designed for the [Fairphone 2][fairphone2].
+
+<a href="https://f-droid.org/packages/community.fairphone.clock/" target="_blank">
+  <img src="https://f-droid.org/badge/get-it-on.png" alt="Get it on F-Droid" height="80"/>
+</a>
+
+## Summary
+
+This clock widget was the default clock in official Android 5 OSes for the Fairphone 2 smartphone, both Fairphone OS and [Fairphone Open][fairphone-open].
+
+This is just a regular clock widget, it doesn't include battery estimation, "Your Fairphone for..." nor Peace of Mind info, for the moment.
+
+Fairphone stopped developing its Fairphone specific apps since the release of Android 6 Marshmallow for the Fairphone 2. Since they were open source, the [WeAreFairphone community](https://github.com/WeAreFairphone) make them available for those users who wanted to keep the original Fairphone experience.
```

Comment:
`they were open source` -> `they are open source`
`want to keep` instead of `wanted to keep`

Reply:
`make` was a typo of mine. I meant `made`. Thanks!